arangod/Cluster/EnsureIndex.h

```diff
@@ -46,10 +46,11 @@ class EnsureIndex : public ActionBase {
   virtual arangodb::Result setProgress(double d) override final;
   virtual bool first() override final;
 
+  static void indexCreationLogging(VPackBuilder& index);
```

Comment:
I think this should either be `VPackBuilder const&` or better `VPackSlice`
This method is not supposed to modify the Builder

Reply:
Using `VPackSlice`: https://github.com/arangodb/arangodb/pull/19699/commits/4e4c2130d309787e698d2d0fc261445f047e27c1